testsuite/python/tests_common.py

```diff
@@ -81,6 +81,7 @@ def func(self):
         # set Parameter
         Inter = interClass(**params)
         Inter.validate_params()
+        Inter._set_params_in_es_core()
```

Comment:
Don't do that. Methods prefixed with `_` are private and should not be called from outside the class.